Sources/Extensions/SpriteKit/SKPhysicsContactExtensions.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SKPhysicsContactExtensions.swift
+//  SwifterSwift
+//
+//  Created by Robbie Moyer on 7/20/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(SpriteKit)
+import SpriteKit
+
+// MARK: - Properties
+public extension SKPhysicsContact {
+    
+    /// SwifterSwift: A convenience variable for accessing the
+    /// nodes involved in a physical contact.
+    public var nodes: [SKNode] {
+        return [bodyA.node, bodyB.node].compactMap({ $0 })
+    }
+}
+
+// MARK: - Methods
+public extension SKPhysicsContact {
+
+    /// SwifterSwift: Checks if a physical contact involves two specific category bitmasks
+    ///
+    ///     if contact.involves(.player, and: .enemy) {
+    ///         // GAME OVER
+    ///     }
+    ///
+    /// - Parameters:
+    ///     - categoryA: The first category to check
+    ///     - categoryB: The second category to check
+    ///
+    /// - Returns: `true` if the collision involves the two specific
+    ///     categories, `false` otherwise.
+    ///
+    public func involves(_ categoryA: UInt32, and categoryB: UInt32) -> Bool {
```

Comment:
I feel like `contains` would be a better name, so it's used quite often in Swift.

Reply:
This one I disagree with. `contains` is indeed used often in Swift, however, in this instance I think saying physical contact *involves* two object is more accurate than saying it *contains* two objects.